imports/plugins/included/notifications/server/methods/notifications.js

```diff
@@ -27,10 +27,10 @@ Meteor.methods({
     const values = {};
     const types = {
       orderCancelled: "Your order was canceled.",
-      newOrder: "Your order is being processed.",
+      newOrder: "You just made an order.",
```

Comment:
I think this phrase is a little awkward. I think "Your order was accepted" or "We've received your order" might work better?